server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/mailbox/MailboxNamespace.java

```diff
@@ -42,6 +42,7 @@
     }
 
     public static MailboxNamespace delegated(String owner) {
+        Preconditions.checkArgument(owner != null);
```

Comment:
`StringUtils.isNullOrEmpty`?

Reply:
Enpty use case is handled the line below by "isBlank" check...

Reply:
sure, but why not doing the two checks at the same time?